src/components/Demographics/Demographics.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { shape } from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  StyleSheet, Text, View, SectionList,
+} from 'react-native';
+
+import {
+  getResources,
+  getPatient,
+  getPatientGender,
+  getPatientBirthDate,
+  getPatientAge,
+  getPatientAddresses,
+  renderAddress,
+} from '../../resources/fhirReader';
+import Colors from '../../constants/Colors';
+import mockBundle from '../../../assets/mock_data/bundle-blake-eichmann.json';
+import { clearPatientData } from '../../features/patient/patientDataSlice';
+
+const Demographics = ({
+  patientData,
+}) => {
+  const resources = patientData ? getResources(patientData) : getResources(mockBundle);
+
+  const patient = getPatient(resources);
+  const birthDate = getPatientBirthDate(patient);
+  const age = getPatientAge(patient);
+  const gender = getPatientGender(patient);
+  const address = renderAddress(getPatientAddresses(patient));
+
+  const demographics = [
+    {
+      title: 'Birth date',
+      data: [birthDate],
+    },
+    {
+      title: 'Age',
+      data: [age],
+    },
+    {
+      title: 'Gender',
+      data: [gender],
+    },
+    {
+      title: 'Address',
+      data: [address],
+    },
+  ];
+
+  // eslint-disable-next-line react/prop-types
+  const Item = ({ title }) => (
+    <View style={styles.demographicsRow}>
+      <Text style={styles.title}>{title}</Text>
+    </View>
+  );
+
+  return (
+    <View style={styles.demographicsContainer}>
+      <View style={styles.panelHeader}>
+        <Text style={styles.panelText}>
+          Demographics
+        </Text>
+      </View>
+      <SectionList
+        sections={demographics}
```

Comment:
I'm getting a warning about using this inside the `SummaryScreen` `ScrollView`. See below:

![image](https://user-images.githubusercontent.com/45667486/108893960-addc8800-75df-11eb-8ebd-0beab8c3161e.png)

Can read more about it here:
https://nyxo.app/fixing-virtualizedlists-should-never-be-nested-inside-plain-scrollviews

I think in this case, since we have a fixed number of `Item`, a map should do it.

Reply:
I see - I was using what's semantically appropriate but looks like the framework has a wart. Will fix!

Reply:
For the time being it works fine - and the client won't see it - so I'll skip muddling with the little details